classes/Product.php

```diff
@@ -1123,26 +1123,6 @@ public function validateField($field, $value, $id_lang = null, $skip = [], $huma
         return parent::validateField($field, $value, $id_lang, $skip, $human_errors);
     }
 
-    /**
```

Comment:
I hesitate to flag this as a BC break, my guess is this was once required because the FO didn't handle the redirection independently or correctly But it is absolutely not required/useful to day and it prevents from correctly setting the expected redirection behaviour

So it's definitely a bug, should we flag it as a BC in case some some developers expect this method to update those fields?

Reply:
It will break https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/admin/AdminSearchController.php#L75 right ?

Reply:
No because `toggleStatus` already exist in ObjectModel, so removing this function reverts it back to the initial purpose: toggling status ^^
It doesn't modify other fields any more

Reply:
@jolelievre I would not call this "a BC break" but rather "a notable change".
We could add a little not in "1.7.8.0 notable changes: toggling the status of product does not modify its redirection settings, this has to be done somewhere else in the code"